src/main/webapp/app/exam/manage/exam-management.route.ts

```diff
@@ -67,9 +67,9 @@ export class ExerciseGroupResolve implements Resolve<ExerciseGroup> {
      * @param route Contains the information about the route to be resolved
      */
     resolve(route: ActivatedRouteSnapshot): Observable<ExerciseGroup> {
-        const courseId = route.params['courseId'] ? route.params['courseId'] : null;
-        const examId = route.params['examId'] ? route.params['examId'] : null;
-        const exerciseGroupId = route.params['exerciseGroupId'] ? route.params['exerciseGroupId'] : null;
+        const courseId = route.params['courseId'] ? route.params['courseId'] : undefined;
+        const examId = route.params['examId'] ? route.params['examId'] : undefined;
+        const exerciseGroupId = route.params['exerciseGroupId'] ? route.params['exerciseGroupId'] : undefined;
```

Comment:
This can be simplified.
```suggestion
        const courseId = route.params['courseId'] || undefined;
        const examId = route.params['examId'] || undefined;
        const exerciseGroupId = route.params['exerciseGroupId'] || undefined;
```